app/views/assessment/AssessmentStart.js

```diff
@@ -18,12 +18,12 @@ import { Colors } from '../../styles';
 import { Icons, Images } from '../../assets';
 
 /** @type {React.FunctionComponent<{}>} */
-export const Start = ({ navigation }) => {
+export const AssessmentStart = ({ navigation }) => {
   let { t } = useTranslation();
   return (
     <Info
       ctaAction={() => {
-        navigation.push('Question', {
+        navigation.push('AssessmentQuestion', {
```

Comment:
should we use the Screens from navigation here?
```suggestion
        navigation.push(Screesn.AssessmentQuestion, {
```